src/run-vehicledatabroker.sh

```diff
@@ -29,9 +31,10 @@ then
 fi
 
 docker run \
+    `if [ ! "$VSPEC_FILE_PATH" == null ]; then echo "-v $VSPEC_FILE_PATH:$VSPEC_FILE_PATH -e KUKSA_DATA_BROKER_METADATA_FILE=$VSPEC_FILE_PATH"; fi` \
```

Comment:
In command IF won't work in bash. You need to start out with an empty variable which you only fill with content, if there is a VSPEC_FILE_PATH available.

```
VSPEC_ARGS=""
if [ "$VSPEC_FILE_PATH" != "" ]; then
  VSPEC_ARGS="-v $VSPEC_FILE_PATH:$VSPEC_FILE_PATH -e KUKSA_DATA_BROKER_METADATA_FILE=$VSPEC_FILE_PATH"
fi

docker run \
  $VSPEC_ARGS \
  ...
```

Reply:
Sure, I tested that... but if you want, we can also take your proposal